### ISSUE TITLE
p1.06

### ISSUE THREAD

**Issue by DarrenCsAcc** [nus-cs2103-AY2223S1/forum#498]
![image](https://user-images.githubusercontent.com/89026703/203781338-fdaf56b1-9f51-4319-af3b-53d7e98376a5.png)

Good day.

Can i check why option 1 is correct but option 4 is not selected? Thank you!

**Comment by maxtance** [comment]
Option 1: The table can be linked to 0 or **1** chair. The table can be linked to **0** or more tables. Hence, option 1 is correct. (Edit: The bolded text suggests why the option is correct.)

Option 4: A table has to be linked with exactly **1** door. Since the object diagram doesn't show an association between table and door, option 4 is incorrect.

**Comment by DarrenCsAcc** [comment]
Thank you! If possible can i check why is option 2 correct since the diagram does not show an association between table and the door? I am guessing it can be 0 and 0 since the option is  * at both ends.

**Comment by riccqi** [comment]
You're right, the * multiplicity can be seen as marking the association optional, as the table can exist without a door.

**Comment by damithc** [comment]
Good answers @maxtance @riccqi
